drivers/backup/portworx/portworx.go

```diff
@@ -2094,6 +2096,21 @@ var (
 	}
 )
 
+var (
+	// KubeVirtRule template contains the template for pre, post rules
+	// and pod selector to be used for kubevirt backup
+	KubevirtRuleTemplate = map[string]map[string]string{
```

Comment:
backup.RuleSpec?

Reply:
Renamed this to KubevirtRuleSpec, for other applications the rules are saved as AppParameters in same file.

Reply:
var KubevirtRuleSpec = map[string]KubevirtRule{
	"default": {
		PreRule: PreRule{
			Rule: RuleSpec{
				ActionList:      []string{"/usr/bin/virt-freezer --freeze --name <vm-name> --namespace <namespace>"},
				PodSelectorList: []string{"vm.kubevirt.io/name=<vm-name>"},
				Background:      []string{"false"},
				RunInSinglePod:  []string{"false"},
				Container:       []string{},
			},
		},
		PostRule: PostRule{
			Rule: RuleSpec{
				ActionList:      []string{"/usr/bin/virt-freezer --unfreeze --name <vm-name> --namespace <namespace>"},
				PodSelectorList: []string{"vm.kubevirt.io/name=<vm-name>"},
				Background:      []string{"false"},
				RunInSinglePod:  []string{"false"},
				Container:       []string{},
			},
		},
	},
}

How about something like this? *RunInSinglePod should have been boolean.

Reply:
As discussed it is better to keep it as is for consistency. Thank you.